doc/classes/Input.xml

```diff
@@ -225,7 +227,7 @@
 			<argument index="0" name="scancode" type="int">
 			</argument>
 			<description>
-				Returns true or false depending on whether the key is pressed or not. You can pass KEY_*, which are pre-defined constants listed in [@Global Scope].
+				Returns true if the key is pressed. You can pass KEY_*, which are pre-defined constants listed in [@Global Scope].
```

Comment:
"true" needs [code][/code] tags.

Second sentence:

    See the [code]KEY_*[/code] constants in [@Global Scope].

Reply:
Added code tags to all JOY_* and similar things and got the once thing removed. I so wanted to put the word "just" there as it is in the actual method name, but guidelines has it banned. ;)

Soon there is nothing original in the file.

Reply:
It's not that it's banned, it's that we want to discourage people just reusing the method name as the description.  It doesn't necessarily mean you can't use the word "just".

Reply:
No, the word "just" is literally in the list of banned words :) "Don't ever use these 8 banned words:"

Reply:
Ah, yes, well that's a different meaning of the word "just". Isn't English great?